### PR TITLE
Radio API Mode

### PR DESCRIPTION
**Pull request by jesseli2002**
edoM IPA oidaR

### CODE REVIEW

File: avionics/avionics.ino [hunk 1]
```diff
@@ -152,12 +167,15 @@ void loop()
...
+    static unsigned long radio_old_time = 0;
+    static unsigned long radio_time_interval = 400; //milliseconds
+
+    // static unsigned long tier_one_old_time = 0;
+    // static uint16_t tier_one_interval = 400;
 
     #ifdef NOSECONE
-        static uint16_t tier_two_interval = 2000;
-        static unsigned long tier_two_old_time = 0;
+        // static uint16_t tier_two_interval = 2000;
```
Comment (a-duen) [inline avionics/avionics.ino]: Perhaps remove these comments if we aren't using them anymore

File: avionics/avionics.ino [hunk 2]
```diff
@@ -206,22 +223,23 @@ void loop()
...
-        noseconeTierTwo(bar_data, acc_data, &temp_sensor_data, IMU_data);
-        tier_two_old_time = new_time;
-    }
-    #endif // def NOSECONE
+    // #ifdef NOSECONE
+    // if ( (new_time - tier_two_old_time) >= tier_two_interval ){
+    //     noseconeTierTwo(bar_data, acc_data, &temp_sensor_data, IMU_data);
+    //     tier_two_old_time = new_time;
+    // }
+    // #endif // def NOSECONE
```
Comment (a-duen) [inline avionics/avionics.ino]: Same comment as above ^

File: avionics/avionics.ino [hunk 3]
```diff
@@ -134,7 +149,7 @@ void setup()
   */
 void loop()
 {
-    static unsigned long timestamp;
+    uint32_t timestamp;
```
Comment (a-duen) [inline avionics/avionics.ino]: why change this one at all / why not change all of them?  Does uint32_t make it a static variable?

Comment (jesseli2002) [reply avionics/avionics.ino]: that is a very good question and uint32_t does not make it static
i think the logic is that uint32_t is always going to be 4 bytes, where as unsigned long has just a minimum and technically could change

**Pull request merged**